simtong-application/src/main/kotlin/team/comit/simtong/domain/holiday/usecase/CancelHolidayUseCase.kt

```diff
@@ -31,8 +31,8 @@ class CancelHolidayUseCase(
             ?: throw HolidayExceptions.NotFound()
 
         when (holiday.type) {
-            HolidayType.HOLIDAY -> if (holiday.status == HolidayStatus.COMPLETED) {
-                throw HolidayExceptions.CannotChange("결정된 휴무일는 취소할 수 없습니다.")
+            HolidayType.HOLIDAY -> if (holiday.isCompleted()) {
+                throw HolidayExceptions.CannotChange("결정된 휴무일은 취소할 수 없습니다.")
```

Comment:
휴무일의 상태를 `COMPLETED`로 바꿀 땐 `share()`을 사용하지만 확인할 땐 `isCompleted()`에요
`isCompleted()` 대신 `isShared()` , 또는 `share()` 대신 `complete()`로 일관성을 부여하면 어떨까요?

```suggestion
            HolidayType.HOLIDAY -> if (holiday.isShared()) {
                throw HolidayExceptions.CannotChange("공유된 휴무일은 취소할 수 없습니다.")
```

Reply:
원래 complete()로 했다가 고민하면서 바꿨는데 원래대로 할게유